WORKSPACE

```diff
@@ -117,8 +117,14 @@ sass_repositories()
 # This dependency specifies the version of protobuf that will be used to compile
```

Comment:
Is this naming standard? Do you think it would be beneficial to somehow rename this reference to make it more explicit that this would be used for "compiling" protos (e.g. @com_google_protobuf_compiler, or _generator, or something like this?)

And then the one below could be something like "expect_protobuf_runtime_installed"?

Or perhaps simply mentioning the version? "@com_google_protobuf_v3_19_6" ? This means that we'd need to change it wherever it's referenced/used, but that seems to some extent clearer?

Reply:
The naming is standard - repo names are typically expected to match the name they define in their own `workspace()` rule, which in this case is `com_google_protobuf`, see here: https://github.com/protocolbuffers/protobuf/blob/main/WORKSPACE#L1  The name `@com_google_protobuf` is also expected by other dependencies like gRPC.

For the `expect_X_installed` py_library target, I'd prefer to keep it closely aligned to the name of the actual package, which is just `protobuf`, since that's the convention we already have in place for the other targets of that form.

I've tried to make the distinction between the two clearer with extended comments, and also, after this PR the usages of @com_google_protobuf are limited to a few places that actual compile protos, and `expect_protobuf_installed` will always be used for the runtime.

Reply:
Ok. Makes sense.